aiida_lsmo/workchains/isotherm.py

```diff
@@ -203,6 +183,49 @@ class IsothermWorkChain(WorkChain):
     it also runs a raspa widom calculation for the Henry coefficient.
     """
 
+    parameters_schema = FF_PARAMETERS_VALIDATOR.extend({
+        Required('zeopp_probe_scaling', default=1.0, description="scaling probe's diameter: molecular_rad * scaling"):
+            NUMBER,
+        Required('zeopp_volpo_samples',
+                 default=int(1e5),
```

Comment:
I'm not aware of a way to do this with voluptuous.
However, the field now only accepts ints, i.e. if you forget the `int()` it will raise and you will notice it quickly.